TOOL-BOX: Helm permission issue

When I try to use Helm in the tool-box image I get the following
```
sh-4.4$ helm version
panic: /home/tool-box/.cache/helm/registry/config.json: stat /home/tool-box/.cache/helm/registry/config.json: permission denied

goroutine 1 [running]:
main.newRootCmd(0xc000232500, 0x1be8b60, 0xc0000be008, 0xc0000b0030, 0x1, 0x1, 0x0)
        /home/circleci/helm.sh/helm/cmd/helm/root.go:196 +0xfbe
main.main()
        /home/circleci/helm.sh/helm/cmd/helm/helm.go:68 +0xb4
```

--- a/tool-box/Dockerfile
+++ b/tool-box/Dockerfile
@@ -6,7 +6,7 @@ ENV OC_VERSION=4.4.7 \
     ODO_VERSION=v1.2.1 \
     ANSIBLE_VERSION=2.9 \
     JQ_VERSION=1.6 \
-    HELM_VERSION=v3.2.1 \
+    HELM_VERSION=v3.2.3 \
     TEKTON_VERSION=0.9.0 \
     HOME=/home/tool-box
 
@@ -38,7 +38,7 @@ WORKDIR $HOME
 ADD ./root /
 
 RUN chmod u+x /usr/local/bin/run && \
-    chown -R 1001:1001 $HOME
+    rm -rf $HOME/.cache
 
 USER 1001
 

--- a/tool-box/Dockerfile
+++ b/tool-box/Dockerfile
@@ -24,11 +24,11 @@ RUN mkdir -m 775 $HOME && \
     pip3 install git+https://github.com/ansible/ansible.git@stable-${ANSIBLE_VERSION}
 
 RUN curl --fail -s https://get.helm.sh/helm-${HELM_VERSION}-linux-amd64.tar.gz | tar -xvz && \
-    chmod u+x linux-amd64/helm && mv linux-amd64/helm /usr/local/bin/ && rm -rf linux-amd64 
+    chmod u+x linux-amd64/helm && mv linux-amd64/helm /usr/local/bin/ && rm -rf linux-amd64 && chown -R 1001:1001 $HOME
 
 RUN curl --fail -sL https://github.com/tektoncd/cli/releases/download/v${TEKTON_VERSION}/tkn_${TEKTON_VERSION}_Linux_x86_64.tar.gz | tar --no-same-owner -xvz -C /usr/local/bin/ tkn
 
-RUN curl --fail https://mirror.openshift.com/pub/openshift-v4/clients/ocp/${OC_VERSION}/openshift-client-linux-${OC_VERSION}.tar.gz | tar -C /usr/local/bin/ -xzf -
+RUN curl --fail -sL https://mirror.openshift.com/pub/openshift-v4/clients/ocp/${OC_VERSION}/openshift-client-linux-${OC_VERSION}.tar.gz | tar -C /usr/local/bin/ -xzf -
 
 RUN curl --fail -sL https://mirror.openshift.com/pub/openshift-v4/clients/odo/${ODO_VERSION}/odo-linux-amd64 -o /usr/local/bin/odo && \
     chmod +x /usr/local/bin/odo

--- a/tool-box/Dockerfile
+++ b/tool-box/Dockerfile
@@ -24,7 +24,7 @@ RUN mkdir -m 775 $HOME && \
     pip3 install git+https://github.com/ansible/ansible.git@stable-${ANSIBLE_VERSION}
 
 RUN curl --fail -s https://get.helm.sh/helm-${HELM_VERSION}-linux-amd64.tar.gz | tar -xvz && \
-    chmod u+x linux-amd64/helm && mv linux-amd64/helm /usr/local/bin/ && rm -rf linux-amd64 && chown -R 1001:1001 $HOME
+    chmod u+x linux-amd64/helm && mv linux-amd64/helm /usr/local/bin/ && rm -rf linux-amd64
 
 RUN curl --fail -sL https://github.com/tektoncd/cli/releases/download/v${TEKTON_VERSION}/tkn_${TEKTON_VERSION}_Linux_x86_64.tar.gz | tar --no-same-owner -xvz -C /usr/local/bin/ tkn
 
@@ -37,7 +37,8 @@ WORKDIR $HOME
 
 ADD ./root /
 
-RUN chmod u+x /usr/local/bin/run
+RUN chmod u+x /usr/local/bin/run && \
+    chown -R 1001:1001 $HOME
 
 USER 1001
 

